services/opencollective/opencollective-base.js

```diff
@@ -38,45 +42,49 @@ export default class OpencollectiveBase extends BaseJsonService {
     }
   }
 
-  async fetchCollectiveInfo(collective) {
-    return this._requestJson({
-      schema: collectiveDetailsSchema,
-      // https://developer.opencollective.com/#/api/collectives?id=get-info
-      url: `https://opencollective.com/${collective}.json`,
-      httpErrors: {
+  async fetchCollectiveInfo({ collective, accountType }) {
+    return this._requestGraphql({
+      schema,
+      url: 'https://api.opencollective.com/graphql/v2',
+      query: gql`
+        query account($slug: String, $accountType: [AccountType]) {
+          account(slug: $slug) {
+            name
+            slug
+            members(accountType: $accountType, role: BACKER) {
+              totalCount
+              nodes {
+                tier {
+                  legacyId
+                  name
+                }
+              }
+            }
+          }
+        }
+      `,
+      variables: {
+        slug: collective,
+        accountType,
+      },
+      httpErrorMessages: {
         404: 'collective not found',
       },
```

Comment:
This looks like a hangover for the old Rest API code. It looks like the standard GraphQl error handling catches the not found case and returns an `InvalidResponse` with message `"No collective found with slug..."`. I think you can ditch this (and the mocked test in `opencollective-all.tester.js`).